members/state_wa_gov_jay_inslee.yaml

```diff
@@ -1,58 +1,60 @@
 bioguide: state_wa_gov_jay_inslee
 contact_form:
   method: post
-  action: ""
   usePhantomJsLogging: true
   steps:
     - visit: "https://iqconnect.lmhostediq.com/iqextranet/EForm.aspx?__cid=FSL_WA_GOV&__fid=100021"
     - find:
-      - selector: input#qi_2_first_name
+      - selector: input[id*='first_name']
+    - javascript:
+      - name: ignore browser alert
+        commands: ["window.alert = function() { }"]
```

Comment:
Should include `required: true` here

Reply:
added that plus made the `success` message a bit more specific